doc/man7/EVP_MAC_KMAC.pod

```diff
@@ -0,0 +1,90 @@
+=pod
+
+=head1 NAME
+
+EVP_MAC_KMAC - The KMAC EVP_MAC implementation
+
+=head1 DESCRIPTION
+
+Support for computing KMAC MACs through the B<EVP_MAC> API.
+
+=head2 Numeric identity
+
+B<EVP_MAC_KMAC128> and B<EVP_MAC_KMAC256> are the numeric identities for this
+implementation, and can be used in functions like EVP_MAC_CTX_new_id() and
+EVP_get_macbynid().
+
+=head2 Supported controls
+
+The supported controls are:
+
+=over 4
+
+=item B<EVP_MAC_CTRL_SET_KEY>
+
+This can only be used after B<EVP_MAC_CTRL_SET_MD> has been set.
```

Comment:
Lose this line

Reply:
Also, you should probably specify that this can't be called after `EVP_MAC_init`